scripts/bin/update-packages

```diff
@@ -72,12 +75,12 @@ TERMUX_PACKAGES_DIRECTORIES=$(jq --raw-output 'del(.pkg_format) | keys | .[]' "$
 
 # Main script to:
 # - by default, decide which update method to use,
-# - but can be overrided by build.sh to use custom update method.
-# - For example: see neovim-nightly's build.sh.
+# - but can be overridden by build.sh to use custom update method.
+# - For example: see cmake's build.sh.
```

Comment:
That is a pure cosmetic change, it can be pushed to master directly.

Reply:
We've discussed that previously in the discord and in #23195.
So I'd like to consider this point acknowledged without further need for comment.
If you do want the "cosmetic" changes merged ahead of, and separately from, the rest of this PR.
Please let me know, but I'll mark these review comments here as resolved for the time being to focus on the substantive suggestions.